src/components/About.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+
+class About extends React.Component {
+  render() {
+    return (
+      <div className="container">
+        <div className="dataview-container">
+          <h1 style={{ marginBottom: '0.75em' }}>
+            <b>About the Project</b>
+          </h1>
+          <h3>Purpose</h3>
+          <p>
+            Students at the University of Oklahoma (OU) want the best education
+            possible, which means enrolling in quality courses taught by
+            competent and passionate instructors. To assess the course offerings
+            prior to enrollment, students typically turn to third-party
+            resources, such as{' '}
+            <a href="https://www.ratemyprofessors.com">Rate My Professor</a>.
+            These external services lack adequate reviews for many professors
+            while representing only the students who took the time to contribute
+            a response - typically, these are the students who either loved or
+            hated a course or instructor. With these external tools as the only
+            option, students at OU are forced to either: enroll in courses with
+            inadequate prior knowledge of the course quality, or base their
+            enrollment decisions on highly-biased evaluations. We started this
+            project to provide a superior alternative to these unappealing
+            options.
+          </p>
+          <h3>Project Goal</h3>
+          <p>
+            We wanted to provide a visualization of course reviews that are{' '}
+            <i>validated</i>, <i>extensive</i>, and <i>representative</i> of the
+            enrolled students. Fortunately, verified student evaluations of
+            teaching are administered through OU at <i>eval.ou.edu</i> at the
```

Comment:
student evaluations of courses? idk I think student evaluations of _teaching_ sounds weird.

Reply:
I dig. Changed to "courses and instructors"